config/producer.properties

```diff
@@ -23,7 +23,7 @@ bootstrap.servers=localhost:9092
 # specify the compression codec for all data generated: none, gzip, snappy, lz4, zstd
 compression.type=none
 
-# name of the partitioner class for partitioning events; default partition spreads data randomly
+# name of the partitioner class for partitioning events; default partitioner will try sticking to a partition until the batch is full
```

Comment:
Do we use "events" consistently? Seems like "records" would be more in keeping with the APIs. 

> The default uses "sticky" partitioning logic, which attempts to fill batches in order to improve throughput.